docs/01-get-started.md

```diff
@@ -95,11 +95,26 @@ $ cd mypod/mypod_flutter
 $ flutter run -d chrome
 ```
 
- The flag `-d chrome` runs the app in Chrome, for other run options please see the Flutter documentation.
+The flag `-d chrome` runs the app in Chrome, for other run options please see the Flutter documentation.
 
 :::info
 
-If you run the app on MacOS you will need to add permissions for outgoing connections in your Xcode project. To do this, open the `Runner.xcworkspace` in Xcode. Then check the _Outgoing Connections (Client)_ under _Runner_ > _Signing & Capabilities_ > _App Sandbox_. Make sure to add the capability for all run configurations.
+### iOS Simulator
```

Comment:
Suggestion: Remove the headers from these infor messages.

These show up in the menu to the right.